tests/test_pipelines.py

```diff
@@ -97,10 +100,14 @@ def _check_pipeline(self, pipeline_file):
     def test_ingredients(self):
         """Check that all pipelines in pipelines/_ingredients are tested."""
         glob_str = os.path.join(self.pipeline_path, "_ingredients", "*.yaml")
+        # The *LSST.yaml pipelines are imported by LATISS/LSSTComCam/LSSTCam
+        # and are not to be tested on their own.
         ingredients = set(
-            [os.path.basename(pipeline) for pipeline in glob.glob(glob_str)]
+            [os.path.basename(pipeline) for pipeline in glob.glob(glob_str) if "LSST.yaml" not in pipeline]
```

Comment:
Am I correct then that none of these new pipelines will be tested until the per-instrument pipelines tickets import them, and then those "sub-class" pipelines will be tested?

Reply:
Yes exactly.  That's why the real testing on this ticket comes from the LATISS implementation.  We can't directly test _ingredients pipelines because they don't have an instrument attached.